apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -765,6 +767,10 @@ const FapProposalsAndAssignmentsTable = ({
           options={{
             search: true,
             selection: true,
+            pageSize: Math.min(10, maxPageLength),
+            pageSizeOptions: [5, 10, 20, maxPageLength]
```

Comment:
I was thinking to put even all of this in another variable would be nice. Something like:
```typescript
const pageSizeOptions = [5, 10, 20, maxPageLength]
              .sort()
              .filter((n) => n <= maxPageLength)
```